### ISSUE TITLE
test: MOLA/LOLA DEM import and behavior

### ISSUE THREAD

**Issue by thareUSGS** [USGS-Astrogeology/socet_gxp_dev#11]

**Comment by thareUSGS** [comment]
For Arc ASCII DEM files, File Open, and make sure to set to degrees. This works and creates Socet GXP "dth" files.

**Comment by thareUSGS** [comment]
Test bringing in re-imported DTMs (i.e. Destriped in ISIS).

**Comment by ReddingUSGS** [comment]
Brining in a destriped CTX DEM seems to work fine.  I tried two HiRISE DEMs, and they both gave an error.
Error reading terrain.

I think it might have something to do with the file size.

**Comment by thareUSGS** [comment]
In testing DEM/DTM GeoTiff import in GXP, we found a pattern why sometimes GXP doesn't offer the user to choose whether the input is of type Image or Topography. Basically if the GeoTiff doesn't have a projection set it will assume an image. Thus when importing a DEM/DTM make sure the GeoTIff has a proper projection defined. The reason ahy we continued to have luck with importing ASCII DTMs is because it must ask the user for projection. This might point to why GXP does not support ISIS3 cube import as DEM/DTM since I assume GXP doesn't have ability to read the map projection for an ISIS3 cube. It can read the pixels (but not the metadata). Perhaps something we can have fixed (since GDAL supports reading the map projection).

Currently conversion of a ISIS3 cube to GeoTIff will default to a meters map projection. This may be just fine although in the past we have imported the DEMs in decimal degrees. To facilitate this we can support a simple helper-script to export a decimal degree GDAL VRT header in degrees (for only Simple Cylindrical though). Then that VRT can be converted to GeoTiff for import. Why not just use ASCII -- They can be huge.
